### PR TITLE
refactoring and doc improvment

### PR DESCRIPTION
**Pull request by surli**

### CODE REVIEW

**Comment by surli** [comment]
@fermadeiral FYI I'm doing also some refactoring in this PR, while I'm documenting it.
The idea is that some method names, for example, seems wrong when I'm documenting them: so then I'm renaming them.

**Comment by fermadeiral** [comment]
@surli oh, sh#*, hahaha (I'm already sad thinking on conflicts ^.^). Ok, Simon, that's great. When it's ready, please, mark me as reviewer in this PR.

**Comment by surli** [comment]
> When it's ready, please, mark me as reviewer in this PR.

Hmmmm. Actually I plan to finish it tomorrow before releasing Repairnator. I don't have much time for finishing that.

**Comment by surli** [comment]
> May I review some random parts even that you didn't still finish, then?

Of course !

File: repairnator/repairnator-pipeline/src/main/java/fr/inria/spirals/repairnator/process/inspectors/StepStatus.java [hunk 1]
```diff
@@ -3,6 +3,10 @@
 import fr.inria.spirals.repairnator.process.step.AbstractStep;
 import fr.inria.spirals.repairnator.states.PipelineState;
 
+/**
+ * This class defines how an AbstractStep has been executed.
```
Comment (fermadeiral) [inline repairnator/repairnator-pipeline/src/main/java/fr/inria/spirals/repairnator/process/inspectors/StepStatus.java]: Is it not better to move `StepStatus` to the same package than `AbstractStep` is?

Comment (surli) [reply repairnator/repairnator-pipeline/src/main/java/fr/inria/spirals/repairnator/process/inspectors/StepStatus.java]: I can check that

**Comment by surli** [comment]
@fermadeiral I'll certainly merge this one in few hours. The idea is to merge this one, to check if some other chores are needed for the repo (like broken links, missing readme parts, etc), fix them in a final PR, and when all PR are merged to release Repairnator.

**Comment by fermadeiral** [comment]
> @fermadeiral I'll certainly merge this one in few hours. The idea is to merge this one, to check if some other chores are needed for the repo (like broken links, missing readme parts, etc), fix them in a final PR, and when all PR are merged to release Repairnator.

Ok, Simon. I'm trying to finish the other PR as soon as I can too.

**Comment by surli** [comment]
@fermadeiral for me this is ready to be merged. Tell me if you're ok

**Comment by fermadeiral** [comment]
> @fermadeiral for me this is ready to be merged. Tell me if you're ok

Simon, I believe I will not have the time to review this PR, I'm trying to finish the other one, so I think you could merge it anyways.

**Pull request merged**